src/content/en/showcase/2016/housing.md

```diff
@@ -0,0 +1,65 @@
+project_path: /web/_project.yaml
+book_path: /web/showcase/_book.yaml
+
+{# wf_published_on: 2016-09-26 #}
+{# wf_updated_on: 2016-09-26 #}
+{# wf_featured_image: /web/showcase/2016/housing/card.png #}
+{# wf_featured_snippet: Most Indian users reach the Internet via 2G and 3G networks, so a fast user experience is essential. To decrease load times, Housing.com added Service Workers and streamlined their site to help consumers quickly find what they’re looking for. Users can continue browsing properties and reviewing previous searches, all while offline. #}
+
+# Housing.com increases conversions and lowers bounce rate by 40% with new PWA {: .page-title }
+
+<img src="images/housing/SearchResultsListingPage.png" class="attempt-right">
+
+### Results
+
+<span class="compare-yes"></span> 38% more conversions
+
+<span class="compare-yes"></span> 40% lower bounce rate
+
+<span class="compare-yes"></span> 10% longer average session
+
+<span class="compare-yes"></span> 30% faster page load
+
+## Company
+Housing.com is one of India’s top startups. On track to receive 50 million
+visits this year, it has become one of the leading online real estate
+platforms in India. After building [Housing Go](https://housing.com/), a
+Progressive Web App (PWA), they saw a 38% increase in total conversions
+across browsers. The new PWA also delivered higher-value users, with
+visitors spending 10% longer per session and returning more often.
+
+## Challenges
+Housing.com knew their mobile users woudn’t tolerate slow load times, which
+significantly affect conversions. They measured the impact of users’ average
+page-load time and found that even a one-second improvement brought a
+significant boost in the conversion rate.
```

Comment:
"They measured the impact of users’ average page-load time and found that even a one-second improvement brought a significant boost in the conversion rate."

Can we do pullquotes?